meson.build

```diff
@@ -77,7 +77,7 @@ meson.add_dist_script(dist_script, meson.project_version())
 ################################################################################
 
 cc = meson.get_compiler('c')
-dependencies = [ dependency('libprotobuf-c'), dependency('threads') ]
+dependencies = [ dependency('libprotobuf-c'), dependency('json-c'), dependency('threads') ]
```

Comment:
I would like to keep ESDM dependencies to a bare minimum. If you think you truly need JSON-C for creating a bit of JSON, please wrap it into a configuration option. I personally think that considering we have very easy JSON, we may not need JSON-C.